src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line before the class declaration.